<h1 align="center">💻 Day 19 💻</h1>

<h2>📚 Learnings of the day:</h2>
<p>Today I learned about how to handle Errors and Exceptions using Try, Catch, Throws and Finally.</p>

<h3>1️⃣ Error: </h3>
<p>• The try ... catch statements mark a block of statements to test (try), and specify a response if an exception is thrown. </p>
<p>• The finally clause is executed after the execution of the try block and the catch clause, it is always executed. </p>
<p>• The throw statement throws a user-defined exception. </p>

<h4>❗ Would you like to know more about handling errors and exceptions?</h4>
<p> Access the link below:</p>
<a href="https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Statements/try...catch">
<img src="https://img.shields.io/static/v1?label=FIX&message=ERRORS&color=f0d500&style=for-the-badge"/>
</a>
